Tests/ApolloTests/Interceptors/MultipartResponseSubscriptionParserTests.swift

```diff
@@ -215,7 +306,29 @@ final class MultipartResponseSubscriptionParserTests: XCTestCase {
       """.crlfFormattedData()
     )
 
-    let expectation = expectation(description: "Payload (null) ignored")
+    let expectation = expectation(description: "Null payload ignored")
+    expectation.isInverted = true
+
+    _ = network.send(operation: MockSubscription<Time>()) { result in
+      expectation.fulfill()
+    }
+
+    wait(for: [expectation], timeout: defaultTimeout)
```

Comment:
I hate tests that _have_ to wait for their timeout. They slow down test runs and you never are actually 100% sure if they are actually verifying behavior or just not failing in time. I know sometimes they are unavoidable, but I wonder if we could avoid this...

Is it guaranteed that payloads are processed in the order they are received, even if received in parallel? If so, could we change this test to send 3 payloads: [1, null, 3] and verify that we just get 2 completion callbacks with the correct values?

Reply:
Sure, I can change the test to not have to rely on the timeout; you make a good point.

As for payload order processing, yes I believe they will be processed in the order received. I don't think it's possible to receive payloads in parallel because the network processing queue serial.

Reply:
Done in https://github.com/apollographql/apollo-ios-dev/pull/287/commits/15f0a48cef37b554a3d980b009229068b831f88f.